Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -0,0 +1,256 @@
+
+        results = []
+        for r in rs:
+            result = r.iter_lines()
+            if self.encoding is not None:
+                result = map(
+                    lambda x: x.decode(self.encoding).encode('utf_8'),
+                    result
+                )
+            else:
+                result = map(
+                    lambda x: x.decode('utf_8'),
+                    result
+                )
+            if self.ignore_regex is not None:
+                result = filter(
+                    lambda x: self.ignore_regex.match(x) is None,
+                    result
+                )
+            results.append(result)
+        return results
+
+
+def get_indicator_fields(itype, line, client):
+    attributes = None
+    value = None
+    line = line.strip()
+    if line:
+        if client.indicator is None:
+            indicator = line.split()[0]
+
+        else:
+            indicator = client.indicator['regex'].search(line)
+            if indicator is None:
+                return attributes, value
+            indicator = indicator.expand(client.indicator['transform'])
+        attributes = {}
+        for f, fattrs in client.fields.items():
+            m = fattrs['regex'].search(line)
+
+            if m is None:
+                continue
+
+            attributes[f] = m.expand(fattrs['transform'])
+
+            try:
+                i = int(attributes[f])
+            except Exception:
+                pass
+            else:
+                attributes[f] = i
+        attributes['value'] = value = indicator
+        attributes['type'] = itype
+    return attributes, value
+
+
+def fetch_indicators_command(client, itype):
+    iterators = client.build_iterator()
+    for iterator in iterators:
+        indicators = []
+        for line in iterator:
+            attributes, value = get_indicator_fields(itype, line, client)
+            if value:
+                indicators.append({
+                    "value": value,
+                    "type": itype,
+                    "rawJSON": attributes,
+                })
+        return indicators
+
+
+def get_indicators_command(client, default_indicator_type, args):
+    itype = args.get('indicator_type', default_indicator_type)
+    limit = int(args.get('limit'))
+    indicators_list = fetch_indicators_command(client, itype)
+    entry_result = camelize(indicators_list[:limit])
+    hr = tableToMarkdown('Indicators', entry_result, headers=['Value', 'Type', 'Rawjson'])
+    return hr, {'CSV.Indicator': entry_result}, indicators_list
+
+
+def test_module(client, args):
+    client.build_iterator()
+    return 'ok', {}, {}
+
+
+def feed_main():
+    # Write configure here
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+    client = Client(**params)
+    command = demisto.command()
+    demisto.info('Command being called is {}'.format(command))
```

Comment:
Isn't it problematic for the fetch-indicators scenario?

Reply:
Problematic how?